docs/Room.md

```diff
@@ -165,28 +165,40 @@ This gives the total devil deal percentage for the floor. It doesn't split it in
     -- this code is current for Repentance as of Jan 2024, other versions might have different values
     local game = Game()
     
+    -- without repentogon there's an edge case with the flipped version of tainted lazarus for:
```

Comment:
can you expand the description for these edge cases? does this text mean that if the player does have these items, the calculation provided in this exampel code returns wrong results?